src/_guides/libraries/library-tour.md

```diff
@@ -1177,9 +1177,11 @@ The `then().catchError()` pattern is the asynchronous version of
 #### Chaining multiple asynchronous methods
 
 The `then()` method returns a Future, providing a useful way to run
-multiple asynchronous functions in a certain order. If the callback
-registered with `then()` returns a Future, `then()` returns an
-equivalent Future. If the callback returns a value of any other type,
+multiple asynchronous functions in a certain order. 
+If the callback registered with `then()` returns a Future, 
+`then()` returns a Future that will be completed
```

Comment:
The explanation looks good to me. I just would like to avoid "will be completed". This seems like the rare example of a valid use of _will_, and we use "Future completes" elsewhere, thus... "will complete."

```suggestion
`then()` returns a Future that will complete
```